src/main/java/rs/teslaris/core/indexmodel/UserAccountIndex.java

```diff
@@ -0,0 +1,41 @@
+package rs.teslaris.core.indexmodel;
+
+import javax.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.data.elasticsearch.annotations.Document;
+import org.springframework.data.elasticsearch.annotations.Field;
+import org.springframework.data.elasticsearch.annotations.FieldType;
+import org.springframework.data.elasticsearch.annotations.Setting;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Document(indexName = "user_account")
+@Setting(settingPath = "/configuration/serbian-analyzer-config.json")
+public class UserAccountIndex {
+
+    @Id
+    private String id;
+
+    @Field(type = FieldType.Text, store = true, name = "full_name")
+    private String fullName;
+
+    @Field(type = FieldType.Text, store = true, name = "email")
+    private String email;
+
+    @Field(type = FieldType.Text, store = true, name = "org_unit_name_sr", analyzer = "serbian_simple", searchAnalyzer = "serbian_simple")
+    private String organisationUnitNameSr;
+
+    @Field(type = FieldType.Text, store = true, name = "org_unit_name_other", analyzer = "english", searchAnalyzer = "english")
+    private String organisationUnitNameOther;
+
+    @Field(type = FieldType.Text, store = true, name = "user_role")
+    private String userRole;
+
+    @Field(type = FieldType.Integer, store = true, name = "databaseId", index = false)
```

Comment:
This is not indexed (index=false), but you are using this field in https://github.com/chenejac/TeslaRIS-backend/pull/66/files#diff-6c50e0870e9d3c1ccf89437e430bf9001c880f2f6e111919f0bbb5d612fc26e4R12

Reply:
That was a bug that I found this morning. It is fixed both here and in the other places :smile: